python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -358,6 +358,18 @@ def runPipeline(self, graph, taskFactory, args, butler=None):
         if not butler.run:
             raise ValueError("no output collection defined in data butler")
 
+        # Enable lsstDebug debugging. Note that this is done once in the
+        # main process before PreExecInit and it is also repeated before
+        # running each task in SingleQuantumExecutor (which may not be
+        # needed if `multipocessing` always uses fork start method).
+        if args.enableLsstDebug:
+            try:
+                _LOG.debug("Will try to import debug.py")
+                import debug
+                assert debug  # silence pyflakes
```

Comment:
I think you mean "silence flake8", as that's the linter we use in CI, and I think we'd usually use a `# noqa:<something>` comment on the prevoius line.

Reply:
Indeed, it's a copy&paste from CmdLineTask and I did not give any thought to it. Will fix it with `noqa:F401`.